boot/makebootable_test.go

```diff
@@ -641,6 +643,7 @@ version: 5.0
 	mockSeedGrubenv := filepath.Join(mockSeedGrubDir, "grubenv")
 	c.Check(mockSeedGrubenv, testutil.FilePresent)
```

Comment:
Should this be c.Assert before FileContains checks below (to avoid noise on failure)?

Reply:
done